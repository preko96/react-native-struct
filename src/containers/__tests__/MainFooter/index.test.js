import React from 'react'
import { MainFooter } from '../../MainFooter/'

it('renders without crashing', () => {
	expect(shallow(<MainFooter />)).toMatchSnapshot()
})