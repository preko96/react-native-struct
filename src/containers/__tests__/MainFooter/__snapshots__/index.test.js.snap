// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Styled(Footer)
  onLayout={[Function]}
>
  <Styled(FooterTab)>
    <Styled(Button)
      active={true}
    >
      <Styled(Icon)
        active={true}
        name="user-check"
        type="Feather"
      />
    </Styled(Button)>
    <Styled(Button)>
      <Styled(Icon)
        name="calendar"
        type="Feather"
      />
    </Styled(Button)>
    <Styled(Button)>
      <Styled(Icon)
        name="bar-chart-2"
        type="Feather"
      />
    </Styled(Button)>
    <Styled(Button)>
      <Styled(Icon)
        name="settings"
        type="Feather"
      />
    </Styled(Button)>
  </Styled(FooterTab)>
</Styled(Footer)>
`;
